modules/op-using-custom-scc-to-improve-prod-security.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc
+:_content-type: PROCEDURE
+
+[id="op-using-custom-scc-to-improve-prod-security_{context}"]
+= Using a custom SCC to improve pod security
+
+To improve the security of your builds and runs, you can use custom security context constraints (SCCs).
+
```

Comment:
Are there any prerequisites for this procedure? Maybe that you logged in as a cluster admin? If so, list them in a `.Prerequisites` section before the `.Procedure` section.